trade-finance-manager-api/src/v1/api.js

```diff
@@ -302,7 +302,7 @@ const findFacilitiesByDealId = async (dealId) => {
   }
 };
 
-const updateFacility = async (facilityId, facilityUpdate) => {
+const updateFacility = async (facilityId, facilityUpdate, sessionUser, options = {}) => {
```

Comment:
fwiw around this amount of parameters, I start thinking about whether it should be an object passed in -- especially as this is not just test code. What do you think about this?